src/Analyzers.xml

```diff
@@ -1629,6 +1629,23 @@ public class C
       </Sample>
     </Samples>
   </Analyzer>
+  <Analyzer>
+    <Id>RCS0062</Id>
```

Comment:
Could you please add `<Option Key="arrow_token_new_line" IsRequired="false" />` to RCS1016 and RR0169?